service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <service verb="get" noun="FemalePersons">
+        <description>
+            Person records for all females
+        </description>
+        <out-parameters>
+            <parameter name="orderList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="orderList">
+                <econdition field-name="gender" operator="equals" value="F"/>
```

Comment:
You can skip the use of operator attribute here, as the default is always equals.